README.md

```diff
@@ -4,4 +4,8 @@ This repository is for showing how Git and GitHub work
 
 ## Purpose
 
-Sample website with plenty of files for demos
\ No newline at end of file
+Sample website with plenty of files for demos
+
```

Comment:
Everything above was already here - as opposed to in the video where he said that this file didn't exist... 🥇